deploy/chart/values-external.yaml

```diff
@@ -0,0 +1,56 @@
+# Values for register clusters
+# NOTE: on-cloud nodes created by register clusters will have alibabacloud.com/external=true label
+
+# deployment
+controller:
+  nodeAffinity:
+    requiredDuringSchedulingIgnoredDuringExecution:
+      nodeSelectorTerms:
+      - matchExpressions:
+        - key: type
+          operator: NotIn
+          values:
+          - virtual-kubelet
+        # csi-provisioner should be scheduled to on-cloud nodes
```

Comment:
Do all register clusters have at least one on-cloud node?

Reply:
If no on-cloud nodes exist, they don't need to install csi-provisioner.

Reply:
OK, so they can just install csi-plugin from ACK console.